src/promptflow/CHANGELOG.md

```diff
@@ -8,10 +8,12 @@
 - Add support for image input and output in prompt flow.
 - [SDK/CLI] SDK/CLI will collect telemetry by default, user can use `pf config set telemetry.enabled=false` to opt out.
 - Add `raise_on_error` for stream run API, by default we raise for failed run.
+- Flow as function: consume a flow like a function with parameters mapped to flow inputs.
 
 ### Bugs Fixed
 
 - [SDK/CLI] Fix node test with dict node input will raise "Required input(s) missing".
+- [SDK/CLI] Will use run name as display name when display name not specified(used flow folder name before).
```

Comment:
nit: should have a space here

specified(used

Reply:
Let me fix in next PR.